scripts/microbe_stage/organelle.as

```diff
@@ -161,7 +161,7 @@ class Organelle{
         return result;
     }
 
-    Float3 calculateCenterOffset() const
+    Float3 calculateCenterOffset(SpeciesComponent@ species) const
```

Comment:
Is it possible that this method wouldn't need to care about if this is for bacteria or not? I'd like to not have this method need to know that.

Reply:
it use inside axialToCartesian method so that one need to know if is bacteria or not like cartesianPosition.